src/Components/ArticleList.jsx

```diff
@@ -0,0 +1,26 @@
+import { useEffect, useState } from "react";
+import { getArticleSummaries } from "../utils/articles_api";
+import { ArticleCard } from "./ArticleCard";
+
+export const ArticleList = () => {
+  const [articleSummaries, setArticleSummaries] = useState([{}]);
```

Comment:
I'd recommend changing this to articles, setArticles, it seems a little confusing calling them summaries.